src/cmdline/mod.rs

```diff
@@ -281,6 +281,15 @@ pub struct CommonCustomizeConfig {
     /// including when Ignition is run.
     #[clap(long, value_name = "path")]
     pub network_keyfile: Vec<String>,
+    /// Nmstate file for live & dest
```

Comment:
I would say `nmstate_file` since it's important to state that it's a file to the state not the state embedded at the argument.

Reply:
Done

Reply:
I was suggesting that we use consistent terminology in the help text.  We're currently saying "Nmstate file" in the short help and "Nmstate config" in the long help.

As to the argument name, I think the original `--network-nmstate` (or maybe `--network-nmstate-file` if it fits) is clearer than the updated `--nmstate-file`, since the first one helpfully groups the network configuration arguments together.  However, I'd be interested in other people's thoughts here.

Reply:
What about removing `nmstate` and keeping `--network-config-file` ? is similar to how it's done at openshift installer at the baremetal host [1].

[1] https://github.com/openshift/installer/blob/969378dc498336d1ac29baa89f75324a44467d5d/pkg/types/baremetal/platform.go#L43

Reply:
@bgilbert what is the best name for this ?

Reply:
`--network-nmstate` sounds good, thanks. :+1: `--network-config-file` implies that a particular file format is the canonical one, which isn't the case here.